File: FirmaTransportSocketsC#/Model/Cursa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    public class Cursa : Entity<int>
    {
        public string destinatie { get; set; }
        public DateTime plecare { get; set; }
        public int locuriDisponibile { get; set; }

        public Cursa(int id, string destinatie, DateTime plecare, int locuriDisponibile)
        {
            this.id = id;
            this.destinatie = destinatie;
            this.plecare = plecare;
            this.locuriDisponibile = locuriDisponibile;
        }

        public Cursa(int id)
        {
            this.id = id;

        }

        public Cursa(string destinatie)
        {
            this.destinatie = destinatie;
        }

        public override string ToString()
        {
            return id + " " + destinatie + " " + plecare + " " + locuriDisponibile;
        }
    }
}

File: FirmaTransportSockets/Model/src/main/java/ft/domain/Cursa.java
package ft.domain;


import java.time.LocalDateTime;


public class Cursa extends Entity<Integer> {
    private String destinatie;
    private String plecare;
    private Integer locuriDisponibile;

    public Cursa() {
    }


    public Cursa(String destinatie) {
        this.destinatie = destinatie;
    }

    public Cursa(String destinatie, String plecare) {
        this.destinatie = destinatie;
        this.plecare = plecare;
    }



    public Cursa(Integer locuriDisponibile) {
        this.locuriDisponibile = locuriDisponibile;
    }

    public Cursa(String destinatie, String plecare, Integer locuriDisponibile) {
        this.destinatie = destinatie;
        this.plecare = plecare;
        this.locuriDisponibile = locuriDisponibile;
    }

    public String getDestinatie() {
        return destinatie;
    }

    public void setDestinatie(String destinatie) {
        this.destinatie = destinatie;
    }

    public String getPlecare() {
        return plecare;
    }

    public void setPlecare(String plecare) {
        this.plecare = plecare;
    }

    public Integer getLocuriDisponibile() {
        return locuriDisponibile;
    }

    public void setLocuriDisponibile(Integer locuriDisponibile) {
        this.locuriDisponibile = locuriDisponibile;
    }



    @Override
    public String toString() {
        return "Cursa{" +
                "destinatie='" + destinatie + '\'' +
                ", plecare=" + plecare +
                ", locuriDisponibile=" + locuriDisponibile +
                ", id=" + id +
                '}';
    }
}
